AmplifyTests/CategoryTests/DataStore/CodingKeysTests.swift

```diff
@@ -0,0 +1,26 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+import Amplify
+@testable import AmplifyTestCommon
+
+class CodingKeysTests: XCTestCase {
+
+    func testSchemaHasCorrectColumnName() throws {
+        ModelRegistry.register(modelType: Comment.self)
```

Comment:
I believe that these tests will have side effects on other tests running in the suite.

After we register these models, how do we reset `ModelRegistry`? -- seems like we need to add a tearDown() function with calls `ModelRegistry.reset()`?

Reply:
To fix, we should add something like:
```
override func tearDown() {
        guard let api = Amplify.API as? Resettable else {
            XCTFail("API Needs to reset between tests")
            return
        }
        api.reset(onComplete: { })
        XCTAssert(ModelRegistry.models.isEmpty)
    }
```
It's not ideal that we have to use Amplify.API, but ModelRegistry is internal (and rightfully so).  In any case, checking to make sure models.isEmpty before proceeding is a good way to double check that a side effect of calling Amplify.API is actually reseting the ModelRegistry.